anaconda_anon_usage/tokens.py

```diff
@@ -128,6 +158,7 @@ def all_tokens(prefix=None):
         client_token(),
         session_token(),
         environment_token(prefix),
+        anaconda_token(),
```

Comment:
It's a broad name, are we sure that's forever the token we'll want to call "Anaconda Token"? Wondering if we could name it `anaconda_cloud_token` to make it forward compatible in case we refactor the auth further as part of the hub work?

Reply:
Happy to rename it

Reply:
I guess one argument against that is that we're moving from `.cloud` to `.com`, right? And are we calling it Anaconda Hub in the future? and...